### PR TITLE
update sortBy to orderBy

### PR DESCRIPTION
**Pull request by kelseiv**
https://influxdata.slack.com/archives/C0VAV6N92/p1657764936862999 Andrew originated this request, and I see the parameter was updated in our Flux docs

@sanderson - updated geo files ~and sample data json files~

@jstirnaman, looks like we still have `sortBy` in the api docs -- I imagine we may need to update those to `orderBy` as well?

### CODE REVIEW

File: static/downloads/influxdb-enterprise-monitor-dashboard.json [hunk 1]
```diff
@@ -102,7 +102,7 @@
 				"legend": {},
 				"tableOptions": {
 					"verticalTimeAxis": false,
-					"sortBy": {
+					"orderBy": {
```
Comment (sanderson) [inline static/downloads/influxdb-enterprise-monitor-dashboard.json:105]: These shouldn’t have been replaced.

File: static/downloads/influxdb-oss-monitor-dashboard.json [hunk 2]
```diff
@@ -102,7 +102,7 @@
 				"legend": {},
 				"tableOptions": {
 					"verticalTimeAxis": false,
-					"sortBy": {
```
Comment (sanderson) [inline static/downloads/influxdb-oss-monitor-dashboard.json:105]: These shouldn’t have been replaced

File: static/downloads/kapacitor-monitor-dashboard.json [hunk 3]
```diff
@@ -102,7 +102,7 @@
 				"legend": {},
 				"tableOptions": {
 					"verticalTimeAxis": false,
-					"sortBy": {
+					"orderBy": {
```
Comment (sanderson) [inline static/downloads/kapacitor-monitor-dashboard.json:105]: These shouldn’t have been replaced

**Review by sanderson — CHANGES_REQUESTED** [review]

File: static/downloads/influxdb-enterprise-monitor-dashboard.json (hunk 1 above)
Comment (kelseiv) [reply static/downloads/influxdb-enterprise-monitor-dashboard.json:105]: thanks, @sanderson - will revert!

**Review by sanderson — APPROVED** [review]

**Pull request merged**